client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -388,6 +391,68 @@ def _resolve_repre_path(
 
         return filepath
 
+    def _validate_parents(self, project_name: str, product_item: ProductItem) -> list:
```

Comment:
`_validate_parents` is a misleading name. One could expect that the parents are only checking for existence. This function is also creating parent data. How about to rename it to something like `_ensure_parents_exists` or something like it.